imports/plugins/core/accounts/server/methods/inviteShopMember.js

```diff
@@ -69,15 +69,19 @@ export default function inviteShopMember(options) {
   const currentUser = Meteor.user();
   const currentUserName = getCurrentUserName(currentUser);
   const user = Meteor.users.findOne({ "emails.address": email });
+  const isEmailVerified = user && user.emails && user.emails[0] && user.emails[0].verified;
```

Comment:
Rather than using index 0, it would be more correct to find the item where `obj.address === email`